ktlint-ruleset-standard/src/main/kotlin/com/github/shyiko/ktlint/ruleset/standard/ImportOrderingRule.kt

```diff
@@ -23,8 +24,8 @@ class ImportOrderingRule : Rule("import-ordering") {
             if (children.isNotEmpty()) {
                 val imports = children.filter { it.elementType == IMPORT_DIRECTIVE }
                 val sortedImports = imports.sortedBy { it.text }
-                if (imports != sortedImports) {
-                    emit(node.startOffset, "Imports must be ordered in lexicographic order", true)
+                if (imports != sortedImports || hasTooMuchWhitespace(children)) {
+                    emit(node.startOffset, "Imports must be ordered in lexicographic order in a single group", true)
```

Comment:
why make the distinction `in a single group`? the rule just takes into account alphabetical order and doesn't care anything about groupings, right?

Reply:
I just wanted to be explicit that the rule will not have any grouping. The rule documentation used to say `Separated by a blank line between each major grouping (android, com, junit, net, org, java, javax).`, which it actually never did and shouldn't do. Also with the addition of the `hasTooMuchWhitespace(children)` check it will now actively remove grouping in all cases. 

But I'm ok with removing the addition again.

Reply:
ah, i didn't have the historical context.
seems like `Imports must be ordered in lexicographic order.` is clear, since there is only one group, but i'm still learning about the project.

Reply:
Removed it 918bda5